force-app/main/default/classes/SOQL.cls

```diff
@@ -53,6 +66,15 @@ public with sharing class SOQL {
     //[public | private ] [static] data_type method_name (input parameters) {
     //  The body of the method
     //}
+       Public static Set<Id> GetContactIdsByLastName(){
+        List<Contact> contacts = [SELECT FirstName,LastName 
+        FROM Contact WHERE LastName = 'Smith' OR LastName = 'Jones'];
+        Set<Id> contactId = new Set<Id>();
+        for(Contact c : contacts){
+            contactId.add(c.Id);
+        }
```

Comment:
There is a special constructor for Map<Id, SObject> that takes a List and separates it into a map of the Id to the record itself. You can use this to return the keySet() of a map, which is a bit simpler (and somehow faster) than doing the iteration on lines 72 through 75.